### ISSUE TITLE
Feedback: Not enough understandable explanation for non FLOSS developers?

### ISSUE THREAD

**Issue by sirblackheart** [sirblackheart/elektrasurvey#59]
> Alles in allem eine eher anstrengende Umfrage, vor allem wenn man nichts mit FLOSS Projekten bisher zu tun hatte und sich mit configurations nicht so beschaeftigt, dann ist Vieles von den Begriffen/Technologien die da vorkommen Neuland und man kann sich eher weniger drunter vorstellen oder man kennt die Technologien und kann sichs in dem Zusammenhang eher schwer vorstellen.

Ist schwer die Relevanz eines solchen Feedbacks einzuschätzen, da unsere Zielgruppe ja FLOSS Entwickler sind und einen gewissen Kontakt Konfigurationssystemen gehabt haben sollten.

**Comment by Namoshek** [comment]
> und einen gewissen Kontakt Konfigurationssystemen gehabt haben sollten.

Can we really expect them to have had experience with it? I don't think every FLOSS developer has.
But for the feedback itself, I would say that this is something we have to deal with. It would probably be interesting if the person who wrote the feedback thought he/she could fill out the survey after reading the introduction (i.e. it is misleading). If not, then I don't see a problem.

**Comment by markus2330** [comment]
Yes, I think we need concrete places and terms for improvements, I'll ask.
